packages/app/components/TransferTest.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactElement, useState, useEffect, SetStateAction } from 'react';
```

Comment:
Added react app test for transfer as well, can be repeated for all methods (optional, is slow)